platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -154,14 +154,18 @@ void onPostMapReady() {
   /**
    * Called when the user
    */
-  void onUpdate() {
-    CameraPosition cameraPosition = transform.invalidateCameraPosition();
-    uiSettings.update(cameraPosition);
-    // FIXME introduce update method with camera position
+  void onUpdateRegionChange() {
     trackingSettings.update();
     annotationManager.update();
   }
 
+  void onUpdateFullyRendered() {
+    CameraPosition cameraPosition = transform.invalidateCameraPosition();
```

Comment:
What do you think of moving these lines into a private method?

- It increases the chances that other methods can use a method when the method is finely grained.
- It allows the higher-level methods to read more like a series of comments (public methods should look like pseudo code, without language statements, etc.).